src/lib/components/navbutton.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	export let route: string;
+	export let title: string;
+</script>
+
+<a class="navbutton" href={route}>
+	{title}
+</a>
+
+<style>
+	.navbutton {
+		font-size: 14px;
+		width: 12rem;
+		aspect-ratio: 6/1;
+
+		color: inherit;
+		text-decoration-line: none;
+		border: 2px solid lightskyblue;
+		border-radius: 1rem;
+		background: rgb(235, 235, 235);
+
+		display: grid;
+		place-items: center;
+	}
+	@media (prefers-color-scheme: dark) {
+		.navbutton {
+			background: var(--secondary-color-theme);
+		}
+	}
```

Comment:
If `rgb(235, 235, 235)` is meant to be `--secondary-color-theme: rgb(231, 231, 231)` as defined in the root CSS selector, you can simply this styling:

```suggestion
		background: var(--secondary-color-theme);

		display: grid;
		place-items: center;
	}
```